docs/index.txt

```diff
@@ -66,48 +65,67 @@ This attribute resolution is delayed (currently a hard coded amount of time) unt
 it succeeds. So for example if you Select a menu option and then look for the
 resulting dialog e.g. ::
 
-  app.UntitledNotepad.MenuSelect("File->SaveAs")
-  app.SaveAs.ComboBox5.Select("UTF-8")
-  app.SaveAs.edit1.SetText("Example-utf8.txt")
-  app.SaveAs.Save.Click()
+  app.UntitledNotepad.menu_select("File->SaveAs")
+  app.SaveAs.ComboBox5.select("UTF-8")
+  app.SaveAs.edit1.set_text("Example-utf8.txt")
+  app.SaveAs.Save.click()
 
 At the 2nd line the SaveAs dialog might not be open by the time this line is
 executed. So what happens is that we wait until we have a control to resolve
 before resolving the dialog. At that point if we can't find a SaveAs dialog with
 a ComboBox5 control then we wait a very short period of time and try again,
 this is repeated up to a maximum time (currently 5 seconds!)
 
-This avoid the user having to use time.sleep or a "Wait" function.
+This avoids the user having to use time.sleep or a "wait" function explicitly.
```

Comment:
I'd rephrase it as: "This is to avoid having to use time.sleep or wait functions explicitly"